homeassistant/components/homeassistant/strings.json

```diff
@@ -1,6 +1,7 @@
 {
   "config": {
     "abort": {
+      "reauth_already_in_progress": "Re-authentication flow is already in progress",
```

Comment:
User clicks "CONFIGURE" in the frontend, and we tell them "Re-authentication flow is already in progress".
I don't think it's clear to the user what they need to do in this case?

FWIW, this is what it looks like in frontend now:
![image](https://github.com/user-attachments/assets/1682985a-45c9-43ea-9437-4c3510e36f09)

Note that clicking "RECONFIGURE" opens the in-progress reauth flow.

Reply:
I think that "RECONFIGURE" button is actually pointing to the reauthentication process. Maybe the button should instead be renamed "RE-AUTHENTICATE"

To trigger a reconfiguration process, you need to click on ... next to CONFIGURE

Reply:
> I think that "RECONFIGURE" button is actually pointing to the reauthentication process. Maybe the button should instead be renamed "RE-AUTHENTICATE"

Yeah, that's what I said 👍 
I did confuse the CONFIGURE button for the reconfigure flow instead of the options flow though.